timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -31,6 +32,8 @@
     ARIMA=ARIMAModel,
     AutoARIMA=AutoARIMAModel,
     AutoETS=AutoETSModel,
+    StatsmodelsETS=StatsmodelsETSModel,
```

Comment:
How about we remove `Statsmodels` from both class names and aliases here, and remove `ARIMAModel` from sktime entirely?

Reply:
I agree that dropping `Statsmodels` in the model names would be nice, but I'm 100% sure about how we want to handle the changes in presets between different releases.

How about we keep the `Statsmodels` suffix for now, but soon make another PR where we completely remove the `sktime` dependency and change SM model names to `ETS`, `ARIMA`, etc.?